mm2src/coins/nft/storage/sql_storage.rs

```diff
@@ -103,7 +103,7 @@ fn create_transfer_history_table_sql(chain: &Chain) -> Result<String, SqlError>
     image_domain TEXT,
     token_name TEXT,
     details_json TEXT,
-    PRIMARY KEY (transaction_hash, log_index)
+    PRIMARY KEY (transaction_hash, log_index, token_id)
```

Comment:
What about already created tables? Shouldn't we do a migration for these tables?

Reply:
hmm yeah, need to do it
 
I think it’s worth considering the use of sql client (eg SQLx) or an ORM (eg Diesel) in the future, if we decide to fully refactor KDF. These sql tools natively support migrations.

Reply:
> I think it’s worth considering the use of sql client (eg SQLx) or ORM (eg Diesel) in the future, if we decide to fully refactor KDF. These sql tools natively support migrations.

Do we have an issue open for this? If not, please open one.